src/model/formats/test-data.ts

```diff
@@ -178,12 +178,33 @@ export const EXPECTED_FOREFLIGHT_CONTENTS = {
               centered: false,
             },
             {
-              prompt: 'scary!',
+              prompt: 'scary multiline note!',
               expectation: '',
               type: ChecklistItem_Type.ITEM_WARNING,
               indent: 1,
               centered: false,
             },
+            {
+              prompt: '',
+              expectation: '',
+              type: ChecklistItem_Type.ITEM_PLAINTEXT,
```

Comment:
ITEM_BLANK?

Reply:
> Should this be ITEM_BLANK?

Well, no, this is actually part of my specially designed evil correctness tests.

I split (and later collapse) multiline notes into indented text items. So if there is a completely blank line in a note, it should be translated into an indented text item, not a space. I use space items to denote empty space between items that cannot be checked off like checklist items in ForeFlight.

I'm afraid relaxing this will make the translation much more complicated and not necessarily more obvious to the user. At least nothing will get lost if you do it the "wrong" way, just the formatting will not be what you expect.